extension/cart/getCartItems.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {SDKContext} context
+ * @param {Object} input
+ * @param {function} cb
+ */
+module.exports = function (context, input, cb) {
+  context.storage.device.get('cart', (err, cart) => {
```

Comment:
When adding a variant item to the cart, the variation is not part of the response (missing).
A variation is something like "Color: Red" (with a quantity, not a simple option).